Cocktail/Cocktail/CocktailStruct.swift

```diff
@@ -185,7 +187,7 @@ struct Cocktail: Codable, Hashable {
         case lemon = "레몬"
         case lemonSqueeze = "레몬즙"
         case appleMint = "애플민트"
-        case whippingCream = "휘핑크림"
+        case whippingCream = " 크림"
```

Comment:
```suggestion
        case whippingCream = "크림"
```
- 불필요한 띄어쓰기
- localization에도 추가할 것